docs/data/material/getting-started/templates/dashboard/components/ChartUserByCountry.js

```diff
@@ -100,7 +100,7 @@ export default function ChartUserByCountry() {
   return (
     <Card variant="outlined" sx={{ pb: '8px' }}>
       <CardContent>
-        <Typography component="h2" variant="subtitle2">
+        <Typography variant="subtitle2" component="p">
```

Comment:
I think we should keep `component="h2"` here as that seems semantically correct for these card headers